zebra/zebra_vty.c

```diff
@@ -3914,6 +3914,7 @@ DEFUN (show_zebra,
 		       ipforward_ipv6() ? "On" : "Off");
 	ttable_add_row(table, "MPLS|%s", mpls_enabled ? "On" : "Off");
 	ttable_add_row(table, "EVPN|%s", is_evpn_enabled() ? "On" : "Off");
+	ttable_add_row(table, "Netlink buffer size|%d", rcvbufsize);
```

Comment:
it's not just a "netlink" value though, right - it's used on other OSes - maybe it should be "Kernel socket buffer" instead?